matlab-hello-world-model/README.md

```diff
@@ -0,0 +1,106 @@
+# Matlab Hello World Model
+
+This is a very simple Model written in Matlab to demonstrate the
+process of Dockerising a Matlab Model for upload onto DAFNI.
+
+The Model itself reads a csv file from a Dataset that has been
+uploaded to DAFNI, an example of what the csv file looks like,
+`dataset.csv`, is included in this folder. The Model then reads
+a parameter from the environment variables, parameters values
+are passed to Models via environment variables on DAFNI. The 
+parameter should contain an integer, the Model will add this
+integer to the Matrix that is stored in the csv file. Finally
+the Model will write the new Matrix to a file as an output.
+
+File Structure:
+
+- _[Dockerfile](./Dockerfile)_ - Builds the container that will be
+   run by DAFNI
+- _[model_definition.yaml](./model_definition.yaml)_ - A 
+   machine-readable file used to define the Model.
+   This information will be shown to other users who may wish to 
+   use your Model.
+- _[dataset.csv](./dataset.csv)_ - an example csv file to run
+   the Model with, on DAFNI this file will be provided by a 
+   dataset.
```

Comment:
Dataslot rather than dataset?

Reply:
I mean technically I guess it will be provided by a dataset in a dataslot. I thought it would make more sense to refer to it as a dataset though.